src/main/kotlin/io/github/smiley4/ktorswaggerui/dsl/DocumentedRouteSelector.kt

```diff
@@ -90,6 +91,19 @@ fun Route.get(
     return documentation(builder) { get(body) }
 }
 
+inline fun <reified T : Any> Route.get(
```

Comment:
i think i'd prefer e.g. moving the resource releated methods into their own class instead of supressing/ignoring detekt. That would also make it clearer for others if they are using the "normal" routes or the resource-routes.